csharp/src/Ice/StringUtil.cs

```diff
@@ -602,16 +602,12 @@ public static string UnescapeString(string s, int start, int end, string special
             return l.ToArray();
         }
 
-        public static string JoinString(string[] values, string delimiters)
+        public static string JoinStringProperty(IReadOnlyList<string> values)
```

Comment:
I think this should still have a comment.

Reply:
Added a comment